Source/v3Net/MeetupBot/MeetupBot.cs

```diff
@@ -140,13 +224,17 @@ public static async Task WelcomeUser(string serviceUrl, string memberAddedId, st
                 }
             }
 
-            var welcomeMessageCard = WelcomeNewMemberCard.GetCard(teamName, userThatJustJoined.Name);
-
-            if (userThatJustJoined != null)
+            if (userThatJustJoined == null)
             {
-                Trace.TraceInformation($"Notify User: [{userThatJustJoined.Name}] about addition to the team: [{teamName}]");
-                await NotifyUser(serviceUrl, welcomeMessageCard, userThatJustJoined, tenantId);
+                Trace.TraceWarning($"User {memberAddedId} not found");
```

Comment:
We should return here since the member with memberAddedId is not found to be a part of the team.